Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -2383,6 +2387,20 @@ def condition(self, fragment, stepList, key, hasHarvest):
         'Era' : 'Phase2C17I13M9',
         'ScenToRun' : ['GenSimHLBeamSpot','DigiTrigger','RecoGlobal', 'HARVESTGlobal', 'ALCAPhase2'],
     },
+    '2026D98' : {
+        'Geom' : 'Extended2026D98',
+        'HLTmenu': '@fake2',
+        'GT' : 'auto:phase2_realistic_T25',
+        'Era' : 'Phase2C17I13M9',
+        'ScenToRun' : ['GenSimHLBeamSpot','DigiTrigger','RecoGlobal', 'HARVESTGlobal', 'ALCAPhase2'],
+    },
+    '2026D99' : {
+        'Geom' : 'Extended2026D99',
+        'HLTmenu': '@fake2',
+        'GT' : 'auto:phase2_realistic_T25',
+        'Era' : 'Phase2C17I13M9',
```

Comment:
Shouldn't this be `Phase2C18I13M9` ?
https://github.com/cms-sw/cmssw/blob/db4ff9a6cf2fea967f3d80ca084d195e6f5a17ed/Configuration/Eras/python/Era_Phase2C18I13M9_cff.py#L6

Reply:
I was cloning what it is done in upgradeWorkflowComponents.py for D95 (C17) and D96 (C18), but I am not expert on this

    '2026D95' : {
        'Geom' : 'Extended2026D95',
        'HLTmenu': '@fake2',
        'GT' : 'auto:phase2_realistic_T21',
        'Era' : 'Phase2C17I13M9',
        'ScenToRun' : ['GenSimHLBeamSpot','DigiTrigger','RecoGlobal', 'HARVESTGlobal', 'ALCAPhase2'],
    },
    '2026D96' : {
        'Geom' : 'Extended2026D96',
        'HLTmenu': '@fake2',
        'GT' : 'auto:phase2_realistic_T21',
        'Era' : 'Phase2C17I13M9',
        'ScenToRun' : ['GenSimHLBeamSpot','DigiTrigger','RecoGlobal', 'HARVESTGlobal', 'ALCAPhase2'],
    },

Reply:
Apparently `Phase2C18I13M9` is used only for D94 (=T24+C20+M10+I15+O9+F8), which looks confusing...

Reply:
is there any action needed on my side?

Reply:
Hi,
`Phase2C18I13M9` is used with geometry with HFNose only, so D94 (C20). Another with C17, C18 will use `Phase2C17I13M9`. This PR is good, and we can review the era later. Thanks for the comment.

Reply:
@srimanob can you clarify why the `Phase2C18I13M9`  is used with geometry with  C20 but not C18? Shouldn't it be called `Phase2C20I13M9` instead?

Reply:
Hi @mmusich 
I agree. That is what I said to be follow up later, with @bsunanda